.github/workflows/docker-image-build-push.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: ${{ github.event.inputs.repo_type == 'official' && 'linux/amd64,linux/arm64' || 'linux/amd64' }}
```

Comment:
What is unofficial image?

Reply:
[unofficial images repo](https://hub.docker.com/r/nethermindeth/juno): Used for testing purposes, storing images built from main or other development branches. These images are not ready for production.

[official images repo](https://hub.docker.com/r/nethermind/juno): Contains production-ready-released images that are thoroughly tested and verified. Images are pushed here only when a new release is ready.